src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTape.java

```diff
@@ -219,7 +224,8 @@ public boolean shouldSideBeRendered(final IBlockState blockState, final IBlockAc
     /**
      * @deprecated (Remove this as soon as minecraft offers anything better).
      */
-    @javax.annotation.Nullable
+    @SuppressWarnings(DEPRECATION)
+    @Nullable
     @Deprecated
     @Override
     public AxisAlignedBB getCollisionBoundingBox(final IBlockState blockState, final IBlockAccess worldIn, final BlockPos pos)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)